pandas/core/window.py

```diff
@@ -451,14 +452,19 @@ class Window(_Window):
 
     Parameters
     ----------
-    window : int, or offset
+    window : int, offset, or BaseIndexer subclass
```

Comment:
ideally type as much as possible (again I know that things are not typed now), maybe makes sense to do a pre-cursor to type what we have now